modules/gdscript/language_server/gdscript_extend_parser.cpp

```diff
@@ -32,9 +32,91 @@
 
 #include "../gdscript.h"
 #include "../gdscript_analyzer.h"
+#include "editor/editor_settings.h"
 #include "gdscript_language_protocol.h"
 #include "gdscript_workspace.h"
 
+int get_indent_size() {
+	if (EditorSettings::get_singleton()) {
+		return EditorSettings::get_singleton()->get_setting("text_editor/behavior/indent/size");
+	} else {
+		return 4;
+	}
+}
+
+lsp::Position GodotPosition::to_lsp(const Vector<String> &p_lines) const {
+	lsp::Position res;
+
+	// special case: `line = 0` -> root class (range covers everything)
```

Comment:
```suggestion
	// Special case: `line = 0` -> root class (range covers everything).
```
As per the [comment style guide](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide), same elsewhere